juju/juju
# Bootstrap name

## Please provide the following details to expedite Pull Request review:

### Checklist


----

## Description of change

Allows a user to now set a canonical name for a controller that will be passed through at bootstrap time.

## QA steps

```sh
juju bootstrap localhost testmctest # exec to the local lxd container and check /var/lib/juju/bootstrap-params
juju bootstrap localhost  --config controller-name=testmctest1 # exec to the local lxd container and check /var/lib/juju/bootstrap-params
juju bootstrap localhost controllertest --config controller-name=testmctest1 # should fail with an error
```
